### PR TITLE
[lifelong learning]: S3 storage support

### PR DESCRIPTION
**Pull request by JoeyHwong-gk**
- Add docs and code comment
- S3 storage support
- Reduce parameters for initial
- work with #85 

Signed-off-by: JoeyHwong <joeyhwong@gknow.cn>

### CODE REVIEW

File: examples/lifelong_learning/atcii/README.md [hunk 1]
```diff
@@ -16,15 +16,15 @@ In this example, you can use [ASHRAE Global Thermal Comfort Database II](https:/
 
 
 
-download [datasets](https://kubeedge.obs.cn-north-1.myhuaweicloud.com/examples/atcii-classifier/dataset.tar.gz), including train, evaluation and incremental dataset.
+We provide a well-processed [datasets](https://kubeedge.obs.cn-north-1.myhuaweicloud.com/examples/atcii-classifier/dataset.tar.gz), which including train, evaluation and incremental dataset.
 ```
```
Comment (JimmyYang20) [inline examples/lifelong_learning/atcii/README.md:20]: incremental =>retained ?  different from incremental job

File: examples/lifelong_learning/atcii/train.py [hunk 2]
```diff
@@ -28,13 +28,18 @@ def main():
     train_data.parse(train_dataset_url, label=DATACONF["LABEL"])
     attribute = json.dumps({"attribute": DATACONF["ATTRIBUTES"]})
     early_stopping_rounds = int(
-        Context.get_parameters(
-            "early_stopping_rounds", 100))
+        Context.get_parameters("early_stopping_rounds", 100)
+    )
```
Comment (JimmyYang20) [inline examples/lifelong_learning/atcii/train.py:32]: change `100` to a constant

Comment (JoeyHwong-gk) [reply examples/lifelong_learning/atcii/train.py:32]: In fact, this is a developer-specified hyperparameter and should not be obtained from lib.

File: examples/lifelong_learning/atcii/README.md (hunk 1 above)
Comment (JoeyHwong-gk) [reply examples/lifelong_learning/atcii/README.md:20]: Lifelong Learning retrain should only uses icremental samples

File: examples/lifelong_learning/atcii/README.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
 # Using Lifelong Learning Job in Thermal Comfort Prediction Scenario
 
 This document introduces how to use lifelong learning job in thermal comfort prediction scenario. 
-Using the lifelong learning job, our application can automatically retrains, evaluates, 
-and updates models based on the data generated at the edge.
+Using the lifelong learning job, our application can automatically retrain, evaluate, 
+and update models based on the data generated at the edge.
 
 ##  Thermal Comfort Prediction Experiment
```
Comment (ugvddm) [inline examples/lifelong_learning/atcii/README.md:7]: please add more words to explain `Thermal Comfort Prediction` what it is, what is problem and requirements in this scenario? How to resolve it by lifelong learning?

File: examples/lifelong_learning/atcii/inference.py [hunk 4]
```diff
@@ -31,12 +31,24 @@ def main():
...
+        "method": "TaskMiningByDataAttr",
+        "param": attribute
+    }
+
+    unseen_task_detect = {
+        "method": utd,
+        "param": utd_parameters
+    }
+
     ll_job = LifelongLearning(
```
Comment (ugvddm) [inline examples/lifelong_learning/atcii/inference.py:44]: we still  name it job? what are other guys thoughts?

Comment (JoeyHwong-gk) [reply examples/lifelong_learning/atcii/inference.py:44]: Have discussed with other contributers that the job in CRD is named differently at different stages, training is the `job`, and inference will be a `service`. Please let me know if there's any better advice.

Also @llhuii @jaypume @TymonXie @JimmyYang20 @MooreZheng PTAL

File: examples/lifelong_learning/atcii/README.md (hunk 3 above)
Comment (JoeyHwong-gk) [reply examples/lifelong_learning/atcii/README.md:7]: This is a very helpful piece of advice. Which do you think is better, quoting a detailed review or explaining it in a readme that app developers are interested in?

Comment (ugvddm) [reply examples/lifelong_learning/atcii/README.md:7]: write here is ok, 
BTW, the pr name is wrong, please check.

File: examples/lifelong_learning/atcii/inference.py (hunk 4 above)
Comment (ugvddm) [reply examples/lifelong_learning/atcii/inference.py:44]: ok

File: examples/lifelong_learning/atcii/README.md (hunk 3 above)
Comment (JoeyHwong-gk) [reply examples/lifelong_learning/atcii/README.md:7]: this pr will be closed because this PR has caused confusion, stay up-to-date by following #143 and #142
